plugins/systems/input/Input.cpp

```diff
@@ -37,15 +37,15 @@ void Systems::InputSystem::shootAction(Engine::GameEngine &engine, size_t entity
             auto &positionComponents = reg.componentManager().getComponents<Components::Position>();
             auto &position = positionComponents[entityIndex];
 
-            int projectilePosX = position->x;
-            int projectilePosY = position->y;
-            int projectileVelX = gun->bulletVelocity;
-            int projectileVelY = 0;
-            int projectileColliderWidth = 10;
-            int projectileColliderHeight = 10;
-            int projectileDamage = gun->bulletDamage;
-            enum Components::TypeID type = Components::TypeID::ALLY_PROJECTILE;
-            std::string spriteId = gun->spriteId;
+                int projectilePosX = position->x;
```

Comment:
Why are you adding an indentation ?

Reply:
This was a mistake when modifying the variables. It has now been removed.

Reply:
:+1: